install.js

```diff
@@ -25,14 +25,14 @@ if (process.env.NPM_CONFIG_PUPPETEER_SKIP_CHROMIUM_DOWNLOAD || process.env.npm_c
   return;
 }
 
-const Downloader = require('./lib/Downloader');
-const downloader = Downloader.createDefault();
+const downloadHost = process.env.PUPPETEER_DOWNLOAD_HOST || process.env.npm_config_puppeteer_download_host;
+
+const puppeteer = require('./index');
```

Comment:
Does this work on node 6???dddddd

Reply:
Of course! the `index.js` is the one that checks for async/await support and picks the right puppeteer.

Reply:
What happens if you checkout from the GitHub and run npm install? Do we built before downloading somewhere?

Reply:
@JoelEinbinder yes, we build in the very first line of the script